src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/model/LimitedPartnershipSubmissionDtoValidationTest.java

```diff
@@ -13,6 +13,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
I think you need a new unit test similar to this one that tests your General Partner DTO JavaBeans validation annotations.

Reply:
Happy that this is covered by the Web MVC tests. As long as we have coverage.